visualisation/backend/utils.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef SERENITY_VISUALISATION_UTILS_HPP
+#define SERENITY_VISUALISATION_UTILS_HPP
+
+#include <iostream>
+#include <iomanip>
+#include <string>
+
+#include "glog/logging.h"
+
+#include "stout/try.hpp"
+
+namespace mesos {
+namespace serenity {
+
+class Precision {
```

Comment:
Can't we rely on the host's clock?

Also, can we use C++11 chrono instead of introducing our own time abstraction?